console/views/store.py

```diff
@@ -124,6 +124,8 @@ def store(page):
         store_data = get_filtered_responses(
             audited_logger, valid, tx_id, ru_ref, survey_id, datetime_earliest, datetime_latest)
     else:
+        # If validation unsuccessful, pretend it was an empty search to give user
```

Comment:
I think if you merge merge-new-console branch into this branch, this'll disappear